src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -109,6 +109,14 @@ public void periodic() {
     SmartDashboard.putNumber("Field pose Yaw", fieldpose[5]);
   }
 
+  public void setLEDsOn() {
```

Comment:
Looks good to me. I wish there were an easy way to toggle lights from both SmartDashboard and directly from the code, but I didn't find one. It might be worth adding setLEDsFlash() too, but we can do that if we decide to use that state.

One note: I'm not sure how well AprilTag tracking will work with LEDs off, but a) it might work fine in the brighter environment at UNH and b) if we care about tracking tags at all, it will probably only be when we have a note ready to go anyway.